cluster/src/test/scala/org/apache/spark/sql/policy/PolicyTest.scala

```diff
@@ -58,6 +55,7 @@ class PolicyTest extends SnappyFunSuite
   }
 
   override def beforeAll(): Unit = {
+    this.stopAll()
```

Comment:
should be reverted

Reply:
This is needed because I am setting a new property in conf
spark.sql.crossJoin.enabled", "true"
which requires current cluster to stop.

Reply:
Ok. I have added few cleanups mentioned in my PR which shall be raised shortly.